test/integration/imagechange_buildtrigger_test.go

```diff
@@ -528,5 +544,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 		if updatedConfig.Spec.Triggers[tc.triggerIndex].ImageChange.LastTriggeredImageID != "registry:5000/openshift/"+tc.name+":"+tc.tag {
 			t.Fatalf("Expected imageID equal to pull spec, got %#v", updatedConfig.Spec.Triggers[0].ImageChange)
 		}
+
+		ignoreBuilds[newBuild.Name] = ""
```

Comment:
One way to see it is that `struct{}` is a zero-sized type, `string` is not. But that's compiler talk, the best part is that by using an empty struct you're telling the readers of the code that the value stored doesn't matter, but only containment matters.